.travis.yml

```diff
@@ -16,3 +16,6 @@ notifications:
 
 before_install:
   - npm config set //registry.npmjs.org/:_authToken=$NPM_TOKEN
+
+after_success:
+  - if [[ $(node -v) == v10* ]]; then npm install coveralls; ./node_modules/.bin/coveralls < coverage/lcov.info; fi
```

Comment:
v minor: I think this would run faster if `coveralls` is a dev dep that is installed with the `npm ci` command. You will pay the price for the `npm i` setup costs twice during the run.

Reply:
Right - I also considered experimenting `npx coveralls` here too, which could be the best option actually. Maybe I'll try this as a follow-up.